errors/ID2043.md

```diff
@@ -1,8 +1,10 @@
 # Error description
 
-> The specified '{0}' is not valid for this client application.
+> The specified 'redirect_uri' is not valid for this client application.
```

Comment:
{0} represents a dynamic parameter. Currently, the same error message/ID is used not only for `redirect_uri` but also for `grant_type`. If we simply replace `{0}` by one of the two possible values, it will make things confusing.

Instead, we should replace `{0}` by [parameter name] and list the possible values with their most likely reason.

Reply:
What is the best way to find out all the different values?

Would searching for ID2043 in the source code do it?

Reply:
> Would searching for ID2043 in the source code do it?

Yes! https://github.com/openiddict/openiddict-core/search?q=ID2043